.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,130 @@
+name: prepare_release
+
+on:
+  push:
+    branches: ['prepare-release-workflow']
+
+  workflow_dispatch:
+    inputs:
+      new-min-consul-version:
+        description: 'The new minimum version of Consul (no "v" prefix)'
+        required: true
+        type: string
+      new-min-consul-k8s-version:
+        description: 'The new minimum version of Consul K8s (no "v" prefix)'
+        required: true
+        type: string
+
+jobs:
+  update-versions:
+    runs-on: ubuntu-latest
+    env:
+      NEW_CONSUL_REQ: ${{ github.event.inputs.new-min-consul-version || '1.13.0' }}
+      NEW_CONSUL_K8S_REQ: ${{ github.event.inputs.new-min-consul-k8s-version || '0.44.0' }}
```

Comment:
The `|| <value>` is only for the PR workflow mentioned above where no inputs are received. Will remove before merge.